upload-apk.sh

```diff
@@ -3,6 +3,7 @@
 mkdir $HOME/buildApk/ 
 #copy generated apk from build folder and README.md to the folder just created
 cp -R app/build/outputs/apk/app-debug.apk $HOME/buildApk/
+mv $HOME/buildApk/app-debug.apk $HOME/buildApk/app-release.apk 
```

Comment:
This change is reflected in many PRs. I think you should clean your branch.
This change was done by my PR.
Thanks.